api/v1beta1/rollout_types.go

```diff
@@ -44,6 +44,14 @@ const (
 	// RollbackInBatchAnnotation is set to rollout annotations.
 	// RollbackInBatchAnnotation allow use disable quick rollback, and will roll back in batch style.
 	RollbackInBatchAnnotation = "rollouts.kruise.io/rollback-in-batch"
+
+	// PartitionReplicasLimitWithTraffic is set to rollout annotations.
+	// PartitionReplicasLimitWithTraffic represents the maximum percentage of replicas
+	// allowed for a step of partition-style release, with traffic/matches specified.
+	// If a step is configured with a number of replicas exceeding this percentage, the traffic strategy for that step
+	// must not be specified. If this rule is violated, the Rollout webhook will block the creation or modification of the Rollout.
+	// The default limit is set to 30%.
+	PartitionReplicasLimitWithTraffic = "rollouts.kruise.io/partition-replicas-limit"
```

Comment:
can we make this annotation a command line arguments ? and plz comment why setting this limit in code

Reply:
done